lib/mayaUsd/fileio/translators/translatorMaterial.cpp

```diff
@@ -89,10 +143,87 @@ MObject UsdMayaTranslatorMaterial::Read(
     return shadingEngine;
 }
 
+namespace {
+using _UVBindings = std::map<TfToken, TfToken>;
```

Comment:
Should the `_GetUVBindingsFromMaterial()` and `_BindUVs()` functions below also go in this anonymous namespace?